corehq/apps/reports/tasks.py

```diff
@@ -82,7 +81,25 @@ def check_es_index():
         notify_exception(None, message='\n'.join(message))
 
 
-@task
+def send_delayed_report(report):
+    """
+    Sends a scheduled report, via  celery background task.
+    """
+    send_report.apply_async(args=[report._id], queue=get_report_queue(report))
+
+
+def get_report_queue(report):
+    # This is a super-duper hacky, hard coded way to deal with the fact that MVP reports
+    # consistently crush the celery queue for everyone else.
+    # Just send them to their own longrunning background queue
+    from mvp.models import MVP
+    if report.domain in MVP.DOMAINS:
```

Comment:
@biyeun is this the best way to check if something is an mvp domain? it looked like it was mostly only used in management commands